AgentAssistAngular/src/common/components/typeahead/typeahead.component.ts

```diff
@@ -100,7 +109,7 @@ export class TypeaheadComponent implements OnInit {
        // }
         if (value?.length > 0) {        
             $.ajax({
-                url: `${connectionDetails.agentassisturl}/agentassist/api/v1/searchaccounts/autosearch?botId=${connectionDetails.botId}`,
+                url: `${connectionDetails.agentassisturl}/agentassist/api/v1/searchaccounts/autosearch?botId=${this.connectionDetails?.autoBotId ? this.connectionDetails?.autoBotId : connectionDetails.botId}`,
```

Comment:
@PoojaPr-Kore  In the Twilio condition, even though the automation bot id is configured, in the URL autoBotId is coming as undefined in the string format. Which is needed to be handled from the UI while sending the autoBotId, please make the appropriate changes to handle the above scenario. 

https://koreteam.atlassian.net/browse/STAT-7031

in the above-attached ticket please go through the comments